lms/static/sass/discussion/views/_inline.scss

```diff
@@ -22,7 +22,7 @@
                 .icon {
                     position: absolute;
                     @include left(7px);
-                    top: 17px;
+                    top: 16px;
```

Comment:
Nit: why is this 16px? Maybe we should drop the absolute positioning and just let flexbox do its thing.